Modules/Devices/src/Devices.Jobs.IdentityDeletion/IdentityDeletionJobConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Backbone.Infrastructure.EventBus;
+using Backbone.Modules.Devices.Infrastructure.Persistence;
+
+namespace Backbone.Modules.Devices.Jobs.IdentityDeletion;
+public class IdentityDeletionJobConfiguration
+{
+    [Required]
+    public InfrastructureConfiguration Infrastructure { get; set; } = new();
+}
+
+public class InfrastructureConfiguration
+{
+    [Required]
+    public EventBusConfiguration EventBus { get; set; } = new();
+    [Required]
+    public IServiceCollectionExtensions.DbOptions SqlDatabase { get; set; } = new();
```

Comment:
Please, let's finally move these DbOptions into one of our class libraries

Reply:
Please let me know if this is what you had in mind 7694b5e2c032b87002606547f2f771506bc00ba3

Reply:
yes 🎈

Reply:
Hmm, I don't like having such a change hidden in this PR. This only causes conflicts in places that don't necessarily have to be caused. Also, it makes it hard to find and undo the commit once it is merged.
If we actually want to make this refactoring (and I'm not fully convinced yet) we should do it in a separate PR.
Don't get me wrong: I highly appreciate you making small refactorings even if they are not task-related. But this one is too big (for the reasons mentioned above).

Reply:
Agreed, naturally, and for the sake of it I did this in a single commit, which can be rolled back, or cherry-picked somewhere else. I'll roll it back for now.